factory/wsConnectorFactory_test.go

```diff
@@ -14,6 +14,7 @@ func createConfig() config.WebSocketConfig {
 		RetryDuration:      1,
 		WithAcknowledge:    false,
 		BlockingAckOnError: false,
+		Mode:               "client",
```

Comment:
there is a constant for this in the `mx-chain-communication-go` module

Reply:
Right, used that one